users/vsantillo/tap_dances/dances/quick_undo.h

```diff
@@ -0,0 +1,36 @@
+// Tap object to track status of Quick Undo
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

```suggestion
#pragma once

// Tap object to track status of Quick Undo
```